python/ray/data/dataset.py

```diff
@@ -480,6 +481,9 @@ def map_batches(
                 ``pandas.DataFrame``, "pyarrow" to select ``pyarrow.Table``, or
                 ``"numpy"`` to select ``numpy.ndarray`` for tensor datasets and
                 ``Dict[str, numpy.ndarray]`` for tabular datasets. Default is "default".
+            prefetch_batches: The number of batches to fetch ahead of the current batch
```

Comment:
When porting this to the new executor, we should try to consolidate `prefetch_batches` and `prefetch_blocks` into a single `prefetch_batches` argument, where we always prefetch enough blocks to satisfy `prefetch_batches`, which should be simple enough to implement since we have the size for each to-be-fetched block on hand.

Reply:
yep +1!